install.js

```diff
@@ -37,7 +37,7 @@ const installGitIgnore = (packageRoot, task) => {
       .filter(line => !legacyArtefacts.includes(line))
       .join("\n");
     writeFileSync(dest, fileContents);
-    execSync(`echo ".prettierrc\ntsconfig.json" >> ${dest}`);
+    execSync(`echo ".prettierrc\ntsconfig.json\ntslint.json" >> ${dest}`);
```

Comment:
Are people used to reading `dest`? I took a few seconds to figure out that you probably meant `destination`.

Reply:
It's just shorthand for `destination`...